async/go-client/pkg/user.go

```diff
@@ -18,11 +18,13 @@
 package pkg
 
 import (
-	"context"
 	"github.com/apache/dubbo-go/common"
 	"github.com/apache/dubbo-go/protocol"
 	"github.com/apache/dubbo-go/remoting"
 	gxlog "github.com/dubbogo/gost/log"
+)
+import (
```

Comment:
hey, guy, move this import block above as the first import block.

Reply:
fixed.